src/main/java/org/opentripplanner/inspector/vector/KeyValue.java

```diff
@@ -1,7 +1,27 @@
 package org.opentripplanner.inspector.vector;
 
+import java.util.Collection;
+import java.util.stream.Collectors;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
 public record KeyValue(String key, Object value) {
   public static KeyValue kv(String key, Object value) {
     return new KeyValue(key, value);
   }
+  public static KeyValue kv(String key, FeedScopedId value) {
```

Comment:
I'm a bit unfamiliar with this part of the code. Could you help me understand what is this KeyValue class used for? And why do we want to stringify the FeedScopedId instead of using the object itself as the value?

I would like the parameter and the property to be marked `@Nullable` if nulls are expected.

Reply:
I wrote a bit of JavaDoc but here is the explanation:

This is a class that contains values that are then serialized by the vector tiles library and used in the Debug UI. Normally, you would use the values for styling but we mainly just display them in popups, like this:

![image](https://github.com/user-attachments/assets/bd48cb88-0c12-4001-8222-cc7406d366ef)

Since the vector tiles are by design a very efficient data format they don't support any complex data types so `FeedScopedId` is silently ignored. Since I kept forgetting this, I created a little helper that convert the id to a string.